### DISCUSSION TITLE
IDP.Generic | A Virus Threat?

### DISCUSSION THREAD

**Discussion by pig3onboi** [BEEmod/BEE2-items#4174] [Tech Support / Q&A] [answered]
(I'm new to this website so just let me know if I am doing anything wrong here)
So I think I have a general idea on what IDP.Generic is, but I'm not exactly sure what to do. I was going to play my level to see how it looked, so I had to go through the process of it building the level, and right as it finished calculating the lighting, the game stopped and my Avast Free Antivirus said that the portal2.exe file was a threat and the infection was IDP.Generic. I am very cautious, so I just want things to be cleared that this tool is not any sort of virus and my computer will be fine.

Thank You!
**-Pig3on**

**Comment by FrozenDragon0** [comment, marked as answer]
@pig3onboi The tool is not a virus whatsoever the things it does is similar to what viruses do but is does no harm to your PC it just triggers a false alarm on your anti-virus. If you really don't trust the .exe go to this website: https://www.virustotal.com/#/home/upload And put it there and it'll tell you if it gets flagged or not.
